src/api/db/schema.rb

```diff
@@ -190,6 +190,12 @@
     t.index ["target_project_id"], name: "index_bs_request_actions_on_target_project_id"
   end
 
+  create_table "bs_request_actions_seen_by_users", id: false, charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
There is a change missing in the schema. The line `ActiveRecord::Schema[7.0].define(version: ...)` should have also changed when you ran the migration locally. This needs to be committed.

Reply:
Done.